### DISCUSSION TITLE
Select

### DISCUSSION THREAD

**Discussion by LeSe0** [heroui-inc/heroui#700] [Feature requests]
Hi, when you plan to release select component

**Comment by jrgarciadev** [comment]
Hey @LeSe0  for sure, it is in our roadmap, stay tuned 🔥

**Reply by rossanmol** [reply]
Any updates @jrgarciadev ?

**Reply by langziyang** [reply]
will support tree select?

**Reply by renanmachad** [reply]
Exists a PR to this feature buts it's stopped, i do maked a question about it.

**Comment by hoaiphu99** [comment]
Still waiting :D

**Comment by jrgarciadev** [comment]
This component is a top priority for upcoming release, stay tuned 👍🏻
